lib/parse_args.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+const { parseArgs } = require('internal/parse_args/parse_args');
+const { emitExperimentalWarning } = require('internal/util');
+
+emitExperimentalWarning('Parse arguments');
```

Comment:
Maybe this should be 'Argument parsing' or something similar since the warning would currently be:

```
ExperimentalWarning: Parse arguments is an experimental feature. This feature could change at any time
```

Reply:
Done.